src/Pages/Login/Login.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+import './Login.scss';
+import '../../Styles/reset.scss';
+import '../../Styles/common.scss';
+
+class Login extends Component {
+  state = {
+    email: '',
+    password: '',
+  };
+  handleInput = e => {
+    const { name, value } = e.target;
+    this.setState({ [name]: value });
+  };
+
+  inputIdValidator = email => {
+    if (email === '') return '';
+    if (!email.includes('@')) return 'inputFail';
+    if (email.includes('@')) return 'inputSuccess';
+  };
+
+  inputPwValidator = pw => {
+    if (pw === '') return '';
+    if (pw.length > 8) return 'inputSuccess';
+    if (pw.length !== '') return 'inputFail';
+  };
+
+  buttonValidator = (email, pw) => {
+    if (email.includes('@') && pw.length > 8) return 'btnSuccess';
+    return '';
+  };
+
+  goToMain = () => {
+    fetch('http://10.58.6.100:8000/users/signin', {
```

Comment:
주소는 매번 바뀔 수 있기 때문에 src/config.js 등에 상수로 분리해서 사용해주세요
```suggestion
    fetch(`{API_URL}/users/signin` {
```

Reply:
처음 알았습니다 감사합니다👍👍👍👍👍